engine/src/gldrv/winsys.cpp

```diff
@@ -201,35 +202,205 @@ void winsys_warp_pointer(int x, int y) {
 int native_resolution_x;
 int native_resolution_y;
 
+SDL_Rect total_display_bounds{};
+SDL_Rect usable_display_bounds{};
+SDL_Rect total_window_size_in_pixels{};
+SDL_Rect usable_window_size_in_pixels{};
+SDL_Rect total_logical_window_size{};
+SDL_Rect usable_logical_window_size{};
+
+/*---------------------------------------------------------------------------*/
+// pmx-20251026
+bool get_sdl_display_name_by_nr(int screen_number, std::string& screen_name, SDL_DisplayID& id) {
```

Comment:
Better make it get_sdl_display_name_by_number. It's long either way and best practice to no longer shorten names.